src/databases/connector.go

```diff
@@ -0,0 +1,206 @@
+package databases
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+	_ "github.com/lib/pq"
+	"gophermart/src/config"
+	"gophermart/src/entities"
+	"log"
+	"sync"
+)
+
+type Service struct {
+	db *sql.DB
+}
+
+var lock = &sync.Mutex{}
+var dbService *Service
+
+func GetInstance() *Service {
+	if dbService == nil {
+		dbService = &Service{}
+		lock.Lock()
+		defer lock.Unlock()
+		sqlInfo := fmt.Sprintf("host=%v port=%v user=%v "+
+			"password=%v dbname=%v sslmode=disable",
+			config.GetHost(), config.GetPort(), config.GetUser(), config.GetPassword(), config.GetDbName())
+		db, err := sql.Open(config.GetDriverName(), sqlInfo)
+		if err != nil {
+			log.Fatal(err)
```

Comment:
```suggestion
			return err
```
и наверху уже log.Fatal.